man/cpg_model_initialize.3.in

```diff
@@ -167,17 +167,20 @@ struct cpg_address {
 .PP
 where nodeid is a 32 bit unique node identifier, pid is the process ID of the process that has joined/left the group
 or sent the message, and reason is an integer code indicating why the node joined/left the group.
+or sent the message, and reason is an integer code indicating why the node joined/left the group (this value is not
```

Comment:
This line has been duplicated rather than replaced :)

Reply:
Good catch, thank you. Fixed in force pushed commit.